app/src/main/java/fr/free/nrw/commons/profile/leaderboard/UserDetailAdapter.java

```diff
@@ -84,6 +93,23 @@ public void onBindViewHolder(@NonNull UserDetailAdapter.DataViewHolder holder, i
             holder.getContext().getResources().getString(R.string.count_prefix),
             leaderboardResponse.getCategoryCount()));
 
+        //When user tap on avatar shows the toast on how to change avatar
+        //fixing:https://github.com/commons-app/apps-android-commons/issues/47747
+        final AccountManager accountManager = AccountManager.get(username.getContext());
+        final Account[] allAccounts = accountManager.getAccountsByType(BuildConfig.ACCOUNT_TYPE);
+        if (allAccounts.length != 0) {
+            currentLoginUserName = allAccounts[0].name;
+        }
+        if (currentLoginUserName.equals(leaderboardResponse.getUsername())) {
+            avatar.setOnClickListener(new DebouncingOnClickListener() {
+                @Override
+                public void doClick(View v) {
+                    Toast.makeText(v.getContext(),
+                        "To set up your leaderboard avatar, tap Set as avatar in the three-dots menu of any image",
```

Comment:
Would you mind adding the `"` quotes, as seen in the issue description?
Also, please internationalize this string, so that it can be localized into different languages.
Thanks! :-)